docs/rest-api.yml

```diff
@@ -700,7 +745,7 @@ paths:
       summary: Get a service by name
       security:
         - oauth2:
-            - read:services
+            - read:services - read:services:name - read:roles:services
```

Comment:
Is this meant to be a list- if so it should be split over multiple lines.

Reply:
Good catch, thanks!